graphql_compiler/compiler/emit_sql.py

```diff
@@ -348,7 +341,7 @@ class SQLFoldObject(object):
     # JOIN VertexPrecedingOutput
     # ON ...
     def __init__(self, dialect: DefaultDialect, outer_vertex_table: Alias, primary_key_name: str):
-        """Create an SQLFoldObject with table, type, and join information supplied by the IR.
+        """Create an FoldSubqueryBuilder with table, type, and join information supplied by the IR.
```

Comment:
nit: `an` -> `a`. [Not part of the PR so feel free to leave], but the docstring seems a bit off (`type` especially)...